### DISCUSSION TITLE
実績の訳語について

### DISCUSSION THREAD

**Discussion by denev-rove** [nsk4762jp/OxygenNotIncluded-Japanese#179] [General]
Spaced Out! で登場した新たな実績名について、私なりに改めて考えてみました。
形にできたものから提案してみます。

- `Cluster Conquest`
新たな翻訳案: `小惑星群の制覇`
`Conquest`は本当の軍事行動ではなく、比喩的表現であると思うので、`制覇`という言い方はどうでしょうか。
主要な山を全て踏破したり、駅のスタンプラリーを完了したりすることを`制覇`と呼んだりするのは珍しくないので、より馴染み深い比喩になると思います。
`征服`でも比喩として十分に通用するので、ぶっちゃけ大した違いではないかもしれませんが、一案として述べてみました。

- `Morale High Ground`
新たな翻訳案: `天にも昇る士気`
"High Ground"には「（ロケットで）高高度の上空にいること」「士気が高揚していること」「（慣用表現として）優位な立場にいること」の3つの意味が掛かっていると思います。
ここで主題となるのは「高い場所にいること」「士気が高いこと」の2つで、「優位であること」は、"high ground"という慣用表現の流用という以上の意味はないように思います。特に競う相手が存在するゲームでもないので。
そこで「天にも昇る気持ち」という言い回しをもじって、2つの意味を掛けた上記の案を考えてみました。

こうしたほうがいいとか、今のままでいいというご意見があればよろしくお願いします。

**Comment by nsk4762jp** [comment]
Spaced Out! の実績名は翻訳に苦労したところなので、新案のほうがずっと良さそうです。

`Radical Trip` (ハジケる旅), `That's Rad!` (凄いじゃん！) についても何かいい案があったらよろしくお願いします。

**Comment by denev-rove** [comment]
どちらも気の利いた表現がないか色々と考え中ですが、なかなか難しいですねー。
こういう複数の意味を持たせた言葉の翻訳は大変ですが、とても楽しいです。（Kleiはこういう表現を多用するので好きです）

**Comment by denev-rove** [comment]
`That's Rad!`
提案：
`元気よく、原子力！`
我ながら安易で何のひねりもないダジャレですが、原義のポジティブな感じが良く出ていると思ったので提案します。
ボツ案：
`容赦せん、放射線！`（意味不明）
`原子炉、実現シロ！`（セガサターン、シロ！）

**Comment by nsk4762jp** [comment]
`元気よく、原子力！`。良いですね。

`研究炉を最大容量で{0}サイクル稼働させる。` 実績なのでぴったりだと思いますよ。

**Comment by denev-rove** [comment]
ありがとうございます。
折を見て修正させていただきます。
